src/plugins/list/README.md

```diff
@@ -48,6 +48,23 @@ A list of set-placements for the plugin. Same for "get" and "error"
 
 Plugin specific config.
 
+## Exported functions
+
+The plugin exports a few useful functions:
+
+```c
+int elektraListMountPlugin (Plugin * handle, const char * pluginName, KeySet * pluginConfig, Key * errorKey)
+int elektraListUnmountPlugin (Plugin * handle, const char * pluginName, Key * errorKey)
+Plugin * elektraListFindPlugin (Plugin * handle, const char * pluginName)
+```
+
+`elektraListMountPlugin` can be used to add a new plugin to the config. The placement will be query from the plugin it self (from its
```

Comment:
Did you maybe mean “queried” instead of “query”?

Reply:
As far as I know, “it self” should be written as [one word](https://www.merriam-webster.com/dictionary/itself) in this context.